en/core-api/bank-transfer.md

```diff
@@ -354,27 +593,29 @@ Please add **bank_transfer** parameter during [API Request](/en/core-api/bank-tr
 ```
 <!-- tabs:end -->
 
-Parameter | Type | Required? | Description
+#### VA Number Specification
+
+Parameter | Type | Required | Description
 --- | --- | --- | ---
-BCA `va_number`| String | (optional) | Length should be within 1 to 11.
-BCA `sub_company_code` | String | (optional) | BCA sub company code directed for this transactions. <br>NOTE: Don't use it if you don't know.
-Permata `va_number` | String | (optional) | Length should be 10. Only supported for b2b VA type.
-BNI `va_number` | String | (optional)| Length should be within 1 to 8.
-BRI `va_number` | String | (optional)| Length should be within 1 to 13.
+BCA va_number| String | (Optional) | Length should be within 1 to 11.
+BCA sub_company_code | String | (optional) | BCA sub company code directed for this transactions. <br>NOTE: Don't use it if you don't know.
+Permata va_number | String | (optional) | Length should be 10. Only supported for b2b VA type.
+BNI va_number | String | (optional)| Length should be within 1 to 8.
+BRI va_number | String | (optional)| Length should be within 1 to 13.
```

Comment:
Do not remove the backtick please, for example leave it as: 
```
BCA `va_number`
```

Because `va_number` is the literal attribute names, while BCA is not. It is important to highlight which one is attribute's name.